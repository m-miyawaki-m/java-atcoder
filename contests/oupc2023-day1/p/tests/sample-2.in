3
10 20
1 1 1 1 1 1 1 1 1 1
4 3000
20 24 1 6
10 3000000
20 2 4 1 6 20 2 4 1 6
