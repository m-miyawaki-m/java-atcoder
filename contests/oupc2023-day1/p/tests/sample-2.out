Yes
3 1
4 7
5 7
10 3
2 9
8 9
3 8
6 9
7 1
No
Yes
10 9
4 6
5 4
4 9
9 3
9 1
8 4
1 2
7 9
